NEWS

```diff
@@ -8,6 +8,9 @@ issues]].
 ** New features
 ** Bug fixes
 ** Braille table improvements
+- Emphasis improvements to US EBAE thanks to Benetech.org
+  - Conforms to BANA emphasis guidelines 
+  - Extracted to an include file to reduce duplication in EBAE-based tables
```

Comment:
This last line is not needed. We don't mention such implementation details in the change log.